doc/guides/dts/howtos.rst

```diff
@@ -199,37 +199,39 @@ See :ref:`missing-dt-binding` for troubleshooting.
 Set devicetree overlays
 ***********************
 
-Devicetree overlays are explained in :ref:`devicetree-intro`. The CMake
-variable :makevar:`DTC_OVERLAY_FILE` contains a space- or semicolon-separated
-list of overlays. If :makevar:`DTC_OVERLAY_FILE` specifies multiple files, they
-are included in that order by the C preprocessor.
-
-Here are some ways to set it:
-
-1. on the cmake build command line
-   (``-DDTC_OVERLAY_FILE="file1.overlay;file2.overlay"``)
-#. with the CMake ``set()`` command in the application ``CMakeLists.txt``,
-   before including zephyr's :file:`boilerplate.cmake` file
-#. create a ``boards/<BOARD>_<revision>.overlay`` file in the application
-   folder for the current board revision. This requires that the board supports
-   multiple revisions, see :ref:`porting_board_revisions`.
-   The ``boards/<BOARD>_<revision>.overlay`` file will be merged with
-   ``boards/<BOARD>.overlay`` if this file also exists.
-#. create a ``boards/<BOARD>.overlay`` file in the application
-   folder, for the current board
-#. create a ``<BOARD>.overlay`` file in the application folder
-#. create an ``app.overlay`` file in the application folder
-
-Here is an example :ref:`using west build <west-building-dtc-overlay-file>`.
-However you set the value, it is saved in the CMake cache between builds.
+You can set devicetree overlays that the C preprocessor must include in the
+build using one of the following methods, listed in order of priority. The
+first method detected by the C preprocessor will make it ignore all the
+later methods in the list.
```

Comment:
We are writing this in a way that makes the user believe the C preprocessor is used to implement the list of mutually exclusive methods.

Reply:
@SebastianBoe Please check if the wording is now more accurate.